docs/plugins/inputs/redis.asciidoc

```diff
@@ -103,40 +103,31 @@ The hostname of your Redis server.
 The name of a Redis list or channel.
 
 [id="plugins-{type}s-{plugin}-password"]
-===== `password`
+===== `password` 
 
   * Value type is <<password,password>>
   * There is no default value for this setting.
 
 Password to authenticate with. There is no authentication by default.
 
 [id="plugins-{type}s-{plugin}-port"]
-===== `port`
+===== `port` 
 
   * Value type is <<number,number>>
   * Default value is `6379`
 
 The port to connect on.
 
-[id="plugins-{type}s-{plugin}-ssl"]
-===== `ssl` 
-
-  * Value type is <<boolean,boolean>>
-  * Default value is `false`
-
-Enable SSL support.
-
-
 [id="plugins-{type}s-{plugin}-threads"]
```

Comment:
REGRESSION: Added with https://github.com/logstash-plugins/logstash-input-redis/pull/61.  Included with 7.2 and 7.3